vng_api_common/inspectors/view.py

```diff
@@ -410,3 +410,15 @@ def _get_response_headers_for_code(self, status_code, direction="response") -> d
             del parameter_type["in"]
             result[parameter.name] = parameter_type
         return result
+
+    def _get_serializer_field_meta(self, field, direction):
+        meta = super()._get_serializer_field_meta(field, direction)
+
+        try:
+            meta["title"] = getattr(
+                field.parent.Meta.model, str(field.source)
+            ).field.verbose_name
+        except (AttributeError, KeyError):
+            meta["title"] = field.field_name
```

Comment:
Voor velden in een serializer kun je natuurlijk ook een `label` meegeven, dit gedrag overschrijf je hier mee. Daarnaast zou ik de title alleen zetten als deze nog niet gezet is.

Reply:
Zonder dit stukje code word het nergens gezet.

Reply:
Maar kan het aanpassen zodat dit alleen effect heeft wanneer title leeg is?

Reply:
> Zonder dit stukje code word het nergens gezet.

De methode die je overschrijft [zet de titel](https://github.com/tfranzel/drf-spectacular/blob/0.23.1/drf_spectacular/openapi.py#L936) als er een `label` meegegeven is aan het veld. Het kan natuurlijk zo zijn dat de velden waarvan je de naam mist geen `label` gezet hebben.